bin/demeteorizer

```diff
@@ -22,6 +22,12 @@ Program
   .option(
     '-j, --json <json>',
     'JSON data to be merged into the generated package.json')
+  .option(
+    '--node-version <version>',
+    'Node version set in generated package.json')
+  .option(
+    '--npm-version <version>',
+    'NPM version set in generated package.json')
```

Comment:
[nitpick] "NPM" should be "npm" and the defaults should be shown in the descriptions.